vendor/DOtherSide/lib/src/DOtherSide.cpp

```diff
@@ -875,7 +875,6 @@ ::DosQObject *dos_qobject_create(void *dObjectPointer, ::DosQMetaObject *metaObj
 void dos_qobject_delete(::DosQObject *vptr)
 {
     auto qobject = static_cast<QObject *>(vptr);
-    qobject->disconnect();
```

Comment:
Good catch @alexjba.

This can potentially cause problems if the object is still in use, for example, if there are pending events for the object in the event queue, leading to crashes.

I believe, this is a bit safer approach: https://github.com/status-im/nimqml/pull/50

Reply:
Nice! Much better! 👍

Reply:
@osmaczko WDYT of keeping this commit even if `dos_qobject_delete` is not used anymore? Just not to leave there code we know can crash the app. `dos_qobject_delete` might look promising to the next developer 😄

https://github.com/status-im/status-desktop/commit/287129490d1b0e9b71ea67b8c71df3f363ca9d75